devU-client/src/components/pages/coursesFormPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React,{useState} from 'react'
+
+import { ExpressValidationError } from 'devu-shared-modules'
+
+import PageWrapper from 'components/shared/layouts/pageWrapper'
+
+import RequestService from 'services/request.service'
+
+//import styles from 'components/pages/coursesFormPage.scss'
+import { useActionless } from 'redux/hooks'
+import TextField from 'components/shared/inputs/textField'
+import Button from 'components/shared/inputs/button'
+import { SET_ALERT } from 'redux/types/active.types'
+
+
+const EditCourseFormPage = () => {
+    const [setAlert] = useActionless(SET_ALERT)
+
+    const [formData,setFormData] = useState({
+        name: '',
+        number: '',
+        semester: '',
+        startDate: '',
+        endDate: '',
+    })
+    const [loading, setLoading] = useState(false)
+
+    const handleChange = (value: String, e : React.ChangeEvent<HTMLInputElement>) => {
+        const key = e.target.id
+        console.log(formData)
+        setFormData(prevState => ({...prevState,[key] : value}))
+    }
+
+    const handleSubmit = () => {
+        setLoading(true)
+
+        RequestService.post('/api/courses/', formData)
+            .then(() => {
+                setAlert({ autoDelete: true, type: 'success', message: 'Course Added' })
+            })
+            .catch((err: ExpressValidationError[] | Error) => {
+                const message = Array.isArray(err) ? err.map((e) => `${e.param} ${e.msg}`).join(', ') : err.message
+
+                setAlert({ autoDelete: false, type: 'error', message })
```

Comment:
This alert is too small to contain error messages and it trims it with ellipses and no option to see the full message. Clicking just removes the message. I know you didn't build this feature and it's not part of this PR, but this should be addressed at some point in the future